### ISSUE TITLE
Potential unclear doc for TLS/UPGRADING

### ISSUE THREAD

**Issue by golgoth** [uswitch/kiam#198]
Hello, 
In the UPGRADING.md, it is written: 

> Server's handling of TLS has changed to remove port from Host. This requires certificates to name kiam-server rather than kiam-server:443, for example. Any issued certificates will likely need re-issuing.

Then, some lines below,
> Generate new TLS assets. You can use docs/TLS.md to create new certificates, or use something like cert-manager or Vault. Given the TLS changes make sure that your server certificate supports names:
kiam-server
kiam-server:443
127.0.0.1

And then, `agent.json` contains:
```
 "hosts": [
        "kiam-server",
        "kiam-server:443",
        "127.0.0.1",
        "127.0.0.1:443",
        "127.0.0.1:9610"
    ], 
```

Am I the only one to find this confusing? I don't get what I will need then for v3.0 in the end :)

**Comment by pingles** [comment]
Yep, thank you. Given the TLS changes for v3, assuming you're _only_ running v3 then you can reduce to

```
 "hosts": [
        "kiam-server",
        "127.0.0.1",
], 
```

👍 for updating the config to reflect this.

**Comment by sbkg0002** [comment]
I tried that @pingles but I was flooded with x509 errors when using v3.2
